src/core/src/FabricGeometry.cpp

```diff
@@ -282,7 +288,8 @@ void FabricGeometry::setTile(
     *worldOrientationFabric = worldOrientation;
     *worldScaleFabric = worldScale;
 
-    displayColorFabric[0] = hasMaterial ? MATERIAL_LOADING_COLOR : DEFAULT_COLOR;
+    displayColorFabric[0] = DEFAULT_VERTEX_COLOR;
```

Comment:
`gltf/pbr.mdl` treats `displayColor` as the default vertex color so I had to change this. Otherwise the color was being multiplied by `(0, 0, 0)` and was being rendered black.

Ideally we would set it to black initially while the material is loading, then white once it's finished, but we can't do that since there's no callback when the material finishes.